HypVINN/README.md

```diff
@@ -54,7 +50,7 @@ Run the HypVINN/run_pipeline.py which has the following arguments:
 ### Usage
 The Hypothalamus pipeline can be run by using a T1 a T2 or both images. 
 Is recommended that all input images are bias field corrected and when passing both T1 and T2 they need to be co-registered.
-The pipeline can do all pre-processing by itself (steps 1 and 2) or omit this step if images are already curated beforehand.
+The pipeline can do all pre-processing by itself (steps 1) or omit this step if images are already curated beforehand.
```

Comment:
```suggestion
The pipeline can do all pre-processing by itself (step 1). This step can be skipped if images are already registered externally. Note, that images are conformed as a first step, which can lead to additional interpolation reducing quality.
```

@santiestrada32 does the registration as part of the pipeline register to the conformed t1? Is there a way we can further resuce the interpolations? 

We don't have to include this in this PR though...